### ISSUE TITLE
Desktop installation fails with the CentOS 7.4 GUI installer

### ISSUE THREAD

**Issue by ronblum** [rstudio/rstudio#2619]
### System details

    RStudio Edition : Desktop
    RStudio Version : RStudio 1.2.522 and 1.1.442
    OS Version      : CentOS 7.4
    R Version       : 3.4.3

### Steps to reproduce the problem

Install RStudio Desktop on CentOS 7.4 by clicking on the link so that it installs with the GUI installer.

### Describe the problem in detail

Installing RStudio Desktop using the GUI installer immediately fails, with the messages "Sorry, this did not work" and "The file is not supported."
![centos_7_4_installer](https://user-images.githubusercontent.com/31009952/38560750-579b4232-3ca4-11e8-9258-868201107be9.png)

### Describe the behavior you expected

RStudio Desktop should install with the GUI, as it does on CentOS 6.9. The first step there is:
![centos_6_9_installer](https://user-images.githubusercontent.com/31009952/38560758-5c73b906-3ca4-11e8-904f-d339cb99e548.png)

Note: 
1) This is not new in 1.2. This is also true for 1.1.442.
2) Installation from the command line works.

**Comment by jmcphers** [comment]
Are there any logs or messages which indicate why the file is "not supported"?

Can you post the output of running this from the command line?

    rpm -ivh rstudio-<version>.rpm

**Comment by ronblum** [comment]
```
rpm -ivh rstudio-1.2.534-x86_64.rpm 
warning: rstudio-1.2.534-x86_64.rpm: Header V4 RSA/SHA1 Signature, key ID e331692f: NOKEY
Preparing...                          ################################# [100%]
Updating / installing...
   1:rstudio-1.2.534-1                ################################# [100%]
```

**Comment by jmcphers** [comment]
Thanks! Next question: does importing the code signing key prior to running the GUI installer make things work?

(not suggesting we'd ask people to do this, just trying to figure out if this is the reason the GUI is rejecting the package)

**Comment by ronblum** [comment]
That works!

**Comment by jmcphers** [comment]
I'm not really sure we can do anything about this. The CentOS 7 GUI installer does not take kindly to unsigned packages, nor packages that don't have a signature it can verify; seasoned CentOS users are going to know how to disable or work around this policy (with e.g. `yum --no-gpg-check`, which is what we recommended for installation on CentOS in years past). The real solution to this is to  set up our own repository with secure metadata (which we're exploring). 

In the meantime the most helpful thing we can do is to tell users what they need to do. I've added this line to our download page:

> Linux users may need to import RStudio's public code-signing key prior to installation, depending on the operating system's security policy.
